staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -677,7 +713,12 @@ func RunTestList(ctx context.Context, t *testing.T, store storage.Interface) {
 				Field: fields.Everything(),
 				Limit: 1,
 			},
-			expectedOut:                []*example.Pod{preset[1]},
+			// TODO(#108003): As of now, watchcache is deliberately ignoring
```

Comment:
So much for genericism :)

Reply:
I'm not proud of it either - if you have other suggestion how to do this in a more generic way that won't be thousands of lines of code, I'm happy to hear and switch to a different approach.